images/modules/com.redhat.osl.cli.packager/install.sh

```diff
@@ -18,19 +18,19 @@ set -e
 mkdir -p "${KN_WORK_DIR}"
 cd "${KN_WORK_DIR}"
 
-mv  "/tmp/LICENSE" .
+mv  "/tmp/artifacts/LICENSE" .
 
 wget -P "${KN_WORK_DIR}" "${KN_ARTIFACT_URL}/amd64/linux/kn-workflow-linux-amd64"
-wget -P "${KN_WORK_DIR}" "${KN_ARTIFACT_URL}/ppc64le/linux/kn-workflow-linux-ppc64le"
-wget -P "${KN_WORK_DIR}" "${KN_ARTIFACT_URL}/s390x/linux/kn-workflow-linux-s390x"
+wget -P "${KN_WORK_DIR}" "${KN_ARTIFACT_URL}/arm64/linux/kn-workflow-linux-arm64"
 wget -P "${KN_WORK_DIR}" "${KN_ARTIFACT_URL}/amd64/windows/kn-workflow-windows-amd64.exe"
 wget -P "${KN_WORK_DIR}" "${KN_ARTIFACT_URL}/amd64/macos/kn-workflow-darwin-amd64"
+wget -P "${KN_WORK_DIR}" "${KN_ARTIFACT_URL}/arm64/macos/kn-workflow-darwin-arm64"
 
-chmod +x kn-workflow-linux-amd64 kn-workflow-linux-ppc64le kn-workflow-linux-s390x kn-workflow-windows-amd64.exe kn-workflow-darwin-amd64
+chmod +x kn-workflow-linux-amd64 kn-workflow-linux-arm64 kn-workflow-windows-amd64.exe kn-workflow-darwin-amd64 kn-workflow-darwin-arm64
 
 tar --transform='flags=r;s|kn-workflow-linux-amd64|kn|' -zcf kn-workflow-linux-amd64.tar.gz kn-workflow-linux-amd64 LICENSE
-tar --transform='flags=r;s|kn-workflow-linux-ppc64le|kn|' -zcf kn-workflow-linux-ppc64le.tar.gz kn-workflow-linux-ppc64le LICENSE
-tar --transform='flags=r;s|kn-workflow-linux-s390x|kn|' -zcf kn-workflow-linux-s390x.tar.gz kn-workflow-linux-s390x LICENSE
+tar --transform='flags=r;s|kn-workflow-linux-arm64|kn|' -zcf kn-workflow-linux-arm64.tar.gz kn-workflow-linux-arm64 LICENSE
 tar --transform='flags=r;s|kn-workflow-darwin-amd64|kn|' -zcf kn-workflow-macos-amd64.tar.gz kn-workflow-darwin-amd64 LICENSE
+tar --transform='flags=r;s|kn-workflow-darwin-arm64|kn|' -zcf kn-workflow-macos-arm64.tar.gz kn-workflow-darwin-arm64 LICENSE
 
 mkdir "${KN_WORK_DIR}/windows" && mv kn-workflow-windows-amd64.exe "${KN_WORK_DIR}/windows/kn.exe" && cp LICENSE "${KN_WORK_DIR}/windows/" && zip --quiet --junk-path - "${KN_WORK_DIR}/windows/*" > kn-workflow-windows-amd64.zip
```

Comment:
The image is failing to build in this part and it seems it is failing when invoking the `zip` command.
 @ricardozanini could you please take a look at it?

Reply:
What's the error?

Reply:
```
+ mkdir /opt/app-root/src/go/src/github.com/knative/client/windows
+ mv kn-workflow-windows-amd64.exe /opt/app-root/src/go/src/github.com/knative/client/windows/kn.exe
+ cp LICENSE /opt/app-root/src/go/src/github.com/knative/client/windows/
+ zip --quiet --junk-path - '/opt/app-root/src/go/src/github.com/knative/client/windows/*'

zip error: Nothing to do! (-)
Error: building at STEP "RUN sh -x /tmp/scripts/com.redhat.osl.cli.packager/install.sh": while running runtime: exit status 12
2024-06-03 18:55:32,012 tools.py:359        ERROR Command '['/usr/bin/podman', 'build', '--squash', '-t', 'openshift-serverless-1/logic-kn-workflow-cli-artifacts-rhel8:1.33.0', '-t', 'openshift-serverless-1/logic-kn-workflow-cli-artifacts-rhel8:latest', '--build-arg=KN_ARTIFACT_URL=http://download.devel.redhat.com/staging-cds/etera/test/kn-workflow-plugin/1/1.33/1.33.0-1/signed/', 'target/image']' returned non-zero exit status 12. Command stdout is 'None' with stderr 'None'
2024-06-03 18:55:32,012 cli.py:542        ERROR Could not run podman ['/usr/bin/podman', 'build', '--squash', '-t', 'openshift-serverless-1/logic-kn-workflow-cli-artifacts-rhel8:1.33.0', '-t', 'openshift-serverless-1/logic-kn-workflow-cli-artifacts-rhel8:latest', '--build-arg=KN_ARTIFACT_URL=http://download.devel.redhat.com/staging-cds/etera/test/kn-workflow-plugin/1/1.33/1.33.0-1/signed/', 'target/image']
Traceback (most recent call last):
  File "/home/rguimara/cekit-4.11.0/lib64/python3.11/site-packages/cekit/tools.py", line 351, in run_wrapper
    result = subprocess.run(
             ^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/usr/bin/podman', 'build', '--squash', '-t', 'openshift-serverless-1/logic-kn-workflow-cli-artifacts-rhel8:1.33.0', '-t', 'openshift-serverless-1/logic-kn-workflow-cli-artifacts-rhel8:latest', '--build-arg=KN_ARTIFACT_URL=http://download.devel.redhat.com/staging-cds/etera/test/kn-workflow-plugin/1/1.33/1.33.0-1/signed/', 'target/image']' returned non-zero exit status 12.
```